src/components/Identity/processIdSyncs.js

```diff
@@ -35,54 +38,60 @@ const setControlObject = obj => {
 
   Object.keys(obj).forEach(id => arr.push(`${id}-${obj[id]}`));
 
-  cookie.set(`${namespace}idSyncControl`, arr.join("_"), {
-    expires: 6 * 30 // 6 months
-  });
+  alloyCookie.set(ID_SYNC_CONTROL, arr.join("_"));
```

Comment:
Are we using hyphens and underscores rather than arrays or objects for optimization purposes? I would typically propose using arrays and objects, but I don't know enough about the context. Is there any risk of `id` or `obj[id]` having a hyphen or underscore in them?

Reply:
Partly for optimization, partly because that's how we had it in the previous libraries. There is no risk.